README.md

```diff
@@ -198,3 +198,84 @@ func Function(
 	
 }
 ```
+
+### Sync Tool and Operator Types
+
+To switch to using just the runtime client and to remove dependencies on individual operators, the sync tool will periodically copy over types from operator repos to this repo. It works by cloning the operator repo, applying changes in the config file, then copying it into this repo.
+
+#### Running
+
+To run the sync tool, use the makefile target.
+
+```
+make lib-sync
+```
+
```

Comment:
We also have config flag which accepts single configuration file. It'll reduce sync time for single package.